src/scss/components/_listings.scss

```diff
@@ -40,9 +41,19 @@ $listing-promoIconColor         : $orange;
     }
 }
 
+.c-listing--preorder,
```

Comment:
Seems weird to be naming things with the function specifically. For example, say we wanted to re-use these styles for a listing component that wasn't for pre-order's or offline restaurants.

Would be better to use modifier that explain the styling they add `c-listing--spaced` (imagine there's a better name out there 😉 )